src/protocols/ssh/settings.c

```diff
@@ -165,6 +166,12 @@ enum SSH_ARGS_IDX {
      */
     IDX_READ_ONLY,
 
+    /**
+     * Number of seconds between sending alive packets.  A default of 0
```

Comment:
I did end up adding some logging to the settings to check when the user enters 1 for the keepalive interval and spit out a warning in the log.  Hopefully this, plus documentation, addresses this concern?